src/onboarding/Start.js

```diff
@@ -430,21 +550,63 @@ const ActionInfo = styled.span`
 
 const Title = styled.h1`
   font-size: 37px;
-  margin-bottom: 40px;
+  margin-bottom: 45px;
+
+  ${breakpoint(
+    'medium',
+    `
+      margin-bottom: 40px;
+    `
+  )}
 `
 
 const OpenOrganization = styled.div`
-  display: flex;
-  flex-direction: column;
+  width: 100%;
+
+  ${breakpoint(
+    'medium',
+    `
+      display: flex;
+      flex-direction: column;
+    `
+  )}
+`
+
+const StyledTextInput = styled(TextInput)`
+  width: 100%;
+  grid-column: 1 / -1;
+
+  ${breakpoint(
+    'medium',
+    `
+      text-align: right;
+    `
+  )}
 `
 
 const Field = styled.div`
-  display: flex;
-  align-items: center;
+  display: grid;
```

Comment:
Wonder if we could simplify this bit. Not opposed to using `grid`, but seems unnecessary when the layout is basically three full-width rows :shrug: